docker.go

```diff
@@ -36,6 +36,9 @@ const (
 
 	// DebianImage is the debian image
 	DebianImage = "debian"
+
+	// FedoraImage is the fedora image
+	FedoraImage = "fedora"
```

Comment:
please add this image here https://github.com/clearcontainers/tests/blob/master/integration/docker/main_test.go#L44

Reply:
@devimc changes applied